src/admin/row/email.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Typography } from '@material-ui/core'
+import PropTypes from 'prop-types'
+
+import { useStyles } from 'admin/row/style'
+import { EmailCell } from 'admin/components/cell/email-cell'
+
+export const Email = ({ user }) => <EmailCell>
+  <Typography className={useStyles().root}>{user.email}</Typography>
```

Comment:
We can write this component as follows:
```js
<Typography classes={useStyles()}>{user.email}</Typography>
```

No need for classnames anymore. :smiley:

Reply:
done